sagemaker-inference-recommender/xgboost/xgboost-inference-recommender.ipynb

```diff
@@ -435,7 +435,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Link: use the page's title for link text rather than a raw link, i.e. <code>Inference Container Images</code></p><p> </p><p>Consistently use <code>XGBoost</code> rather than <code>xgboost</code>,<code>Xgboost</code>. Use <code>SageMaker</code> instead of <code>Sagemaker</code>   </p><p> </p><p>Recommended to use backticks for <code>ModelInput</code> rather than "ModelInput"       </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>